app/models/nomis/offender_base.rb

```diff
@@ -66,18 +63,7 @@ def over_18?
     end
 
     def awaiting_allocation_for
-      omic_start_date = if welsh_offender
-                          ResponsibilityService::WELSH_POLICY_START_DATE
-                        else
-                          ResponsibilityService::ENGLISH_POLICY_START_DATE
-                        end
-
-      if sentence.sentence_start_date.nil? ||
-          sentence.sentence_start_date < omic_start_date
-        (Time.zone.today - omic_start_date).to_i
-      else
-        (Time.zone.today - sentence.sentence_start_date).to_i
-      end
+      (Time.zone.today - reception_date).to_i
```

Comment:
Reception date is only available in offender_summary, and this is only called/used for summary pages, so perhaps might be better in the subclass?